src/webview/direct-connect-form.html

```diff
@@ -90,6 +90,8 @@ <h1>Create a new connection</h1>
                 name="bootstrap_servers"
                 type="text"
                 placeholder="host:port, host2:port2"
```

Comment:
Nit unrelated to this PR I just noticed: should we remove the space between the two servers here?
```suggestion
                placeholder="host:port,host2:port2"
```

Reply:
oh maybe so! Does the space create an error or prevent connection?

Reply:
Nope, looks like it's still functional with comma+space:
<img width="662" alt="image" src="https://github.com/user-attachments/assets/5e7c9035-7f85-44a7-ad9c-66a241ae1fc5">